torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -20,6 +21,42 @@ bool shouldAnnotate(const TypePtr& type) {
 bool shouldAnnotate(const Value* v) {
   return shouldAnnotate(v->type());
 }
+
+std::unordered_set<Node*> getMatchingWaits(Value* fut) {
```

Comment:
haven't taken full look at new changes but It doesn't look like this is being called except recursively

Reply:
blah, this was left in, will remove!